src/envo/envo-edit.owl

```diff
@@ -37272,6 +37302,57 @@ AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <ht
 AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_02000131> "flue gas desulfurization material"@en)
 SubClassOf(<http://purl.obolibrary.org/obo/ENVO_02000131> <http://purl.obolibrary.org/obo/ENVO_02000127>)
 
+# Class: <http://purl.obolibrary.org/obo/ENVO_02000133> (nuclear reactor)
+
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "https://en.wikipedia.org/wiki/Nuclear_reactor") <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_02000133> "a human construction which is used to initiate and control a fission nuclear chain reaction or nuclear fusion reactions.")
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> <http://purl.obolibrary.org/obo/ENVO_02000133> <https://orcid.org/0000-0002-6601-2165>)
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_02000133> "nuclear reactor"@en)
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_02000133> <http://purl.obolibrary.org/obo/ENVO_00000070>)
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_02000133> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0002215> ObjectIntersectionOf(<http://purl.obolibrary.org/obo/ENVO_2000012> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/BFO_0000055> <http://purl.obolibrary.org/obo/ENVO_2000025>))))
+
+# Class: <http://purl.obolibrary.org/obo/ENVO_02000134> (thermal nuclear reactor)
+
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "https://en.wikipedia.org/wiki/Thermal-neutron_reactor") <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_02000134> "a nuclear reactor which uses slow or thermal neutrons.")
```

Comment:
Capitalisation. Further, this definition seems insufficient - what are slow neutrons?

Reply:
Fixed definition